src/GameLogic/Serialization/LevelDeserializer.ts

```diff
@@ -47,6 +48,18 @@ export class LevelDeserializer {
 						data,
 					);
 					break;
+
+				case EntityType.Iceblock:
+					const iceblock = new Iceblock(data.contains);
```

Comment:
I think you'd have to excplicitly serialize and deserialize the contained entity.